docs/developer/backends.rst

```diff
@@ -67,9 +76,16 @@ backend of ``ClientB`` is slow to compute inverse kinematics but can plan motion
 
 .. code-block:: python
 
-    with ClientA() as client_a, ClientB() as client_b:
-        inverse_kinematics = ClientAInverseKinematics(client_a)
-        plan_motion = ClientBPlanMotion(client_b)
+    with ClientA(robot) as client_a, ClientB(robot) as client_b:
+        inverse_kinematics = client_a.planner.inverse_kinematics
+        plan_motion = client_b.planner.plan_motion
+
+        frame = Frame([0, 0, 0], [1, 0, 0], [0, 1, 0])
+        ik_result = inverse_kinematics(frame)
+
+        start_configuration = robot.zero_configuration()
+        goal_configuration = robot.random_configuration()
+        motion_plan = plan_motion(start_configuration, goal_configuration)
```

Comment:
This second example in general has never made much sense. I think we could kill the entire `ClientA`/`ClientB` example.

Reply:
So do we remove the rest starting from "These interface also exist to allow mixing..."?